terraform/services/api-rds/main.tf

```diff
@@ -1,10 +1,48 @@
+locals {
+  db_name = {
+    ab2d = {
+      dev  = "ab2d-dev"
+      test = "ab2d-east-prod-test"
```

Comment:
The ab2d test environment database is "ab2d-east-impl". See my comment above for handling of the ab2d-east-prod-test database.